e2e/pages/adf/searchFiltersPage.ts

```diff
@@ -28,6 +28,8 @@ export class SearchFiltersPage {
     showMoreButtonForSize = this.fileSizeFilter.element(by.css('button[title="Show more"]'));
     showLessButtonForSize = this.fileSizeFilter.element(by.css('button[title="Show less"]'));
     numberOfCheckboxesforSize = element.all(by.css('mat-checkbox[data-automation-id*="checkbox-2:Size"]'));
+    fileTypeFilterHeader = element(by.css('mat-expansion-panel-header[id="mat-expansion-panel-header-7"]'));
```

Comment:
can you use better selectors like [data-automation-id="expansion-panel-1:Type] and so on thanks